deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml

```diff
@@ -135,11 +127,31 @@ Resources:
           EntryPoint:
             - "/bin/sh"
             - "-c"
-            - "while true; do echo 'statsdTestMetric:1|c' | socat -v -t 0 - UDP:127.0.0.1:8125; sleep 1; done"
-      Memory: '2048'
-      Cpu: '1024'
+            - "while true; do echo 'statsdTestMetric:1|c' | socat -v -t 0 - UDP:aws-collector:8125; sleep 1; done"
+          Links:
+            - aws-collector
+        - Name: traffic-generator-xray
+          Image: 'curlimages/curl:latest'
+          Essential: false
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-create-group: 'True'
+              awslogs-group: /ecs/traffic-generator-xray
+              awslogs-region: !Ref 'AWS::Region'
+              awslogs-stream-prefix: ecs
+          DependsOn:
+            - ContainerName: aws-xray-data-emitter
+              Condition: START
+          EntryPoint:
+            - "/bin/sh"
+            - "-c"
+            - "while :; do curl http://aws-xray-data-emitter:5000/outgoing-http-call; sleep 5s; curl http://aws-xray-data-emitter:5000/aws-sdk-call; sleep 10s; done"
+          Links:
+            - aws-xray-data-emitter
       RequiresCompatibilities:
         - EC2
+      Memory: '2048'
```

Comment:
Did the CPU definition get deleted accidentally or was it intentionally removed?

Reply:
cpu added.